static/src/javascripts-legacy/projects/common/modules/experiments/tests/tailor-survey.js

```diff
@@ -48,31 +48,122 @@ define([
             return !(config.page.isAdvertisementFeature) &&
                 config.page.contentType === 'Article'
         };
+        function callTailor(bwid, surveysNotShowAgain) {
+            // If we want to force tailor to show a particular survey we can set an attribute in local storage to have
+            // key = 'surveyToShow', and value = the survey id. Tailor will then override other logic for display, and
+            // look for a survey with this ID to return. This is useful as we can easily see how a particular survey
+            // would be rendered, without actually putting it live. If this parameter is empty or not specified, tailor
+            // behaves as usual.
+            var surveyToShow = localStorage.getItem('surveyToShow');
 
-        function callTailor(bwid) {
-            var endpoint = 'https://tailor.guardianapis.com/suggestions?browserId=' + bwid;
+            var endpoint = 'https://tailor.guardianapis.com/suggestions?browserId=' + bwid + '&edition=' + config.page.edition +
```

Comment:
I would suggest adding `tailorHost` to the Javascript config as you are using it a couple of places in Frontend now. i.e. https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/13-Update-configuration-in-s3.md